### PR TITLE
Adding ksoap2-okhttp module. Keep in mind that, OkHttp supports Andro…

### PR DESCRIPTION
**Pull request by canyapan**
Adding ksoap2-okhttp module. Keep in mind that, OkHttp supports Android 2.3 and above. 
For Java, the minimum requirement is 1.7. (see https://square.github.io/okhttp/ )

You should also add okhttp dependency to your android projects.
 compile 'com.squareup.okhttp3:okhttp:3.9.1'
If you use NTLM authentication, you should also add jcifs.
 compile 'jcifs:jcifs:1.3.17'

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: java
-jdk: oraclejdk7
+jdk: oraclejdk8
```
Comment (mosabua) [inline .travis.yml]: Why Java 8? Is that required now for Android dev and runtime? Fine by me, just want to confirm.

**Comment by mosabua** [comment]
Let me know when this is ready to look at and potentially merge.

File: .travis.yml (hunk 1 above)
Comment (canyapan) [reply .travis.yml]: It is not about using Java 8. Travis-CI is no longer supporting oraclejdk7. Now, they only support oraclejdk8, oraclejdk9 or openjdk7.

please see,
https://github.com/travis-ci/travis-ci/issues/7884#issuecomment-308451879
https://github.com/travis-ci/travis-ci/issues/7019#issuecomment-316769716

Comment (mosabua) [reply .travis.yml]: To be honest I would rather run the build with openjdk7 ... wanna change to that and lets see how that goes? Alternatively I can switch in master and we can see there and have that as a separate commit/change from this PR.

Comment (canyapan) [reply .travis.yml]: It seems, the project compiles well with OpenJDK7, too.

**Comment by canyapan** [comment]
@mosabua, I think, it is safe to call it a beta.

**Pull request merged**